src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -580,6 +547,55 @@ CHIP_ERROR Spake2pVerifier::ComputeWS(uint32_t pbkdf2IterCount, const ByteSpan &
                                 pbkdf2IterCount, ws_len, ws);
 }
 
+CHIP_ERROR ReadDerLength(Reader & reader, size_t & length)
```

Comment:
Since size of size_t differs on 32/64 bit platform this code now yields different results depending on the platform.